src/main/java/frc/team2412/robot/commands/autonomous/JackFiveBallAutoCommand.java

```diff
@@ -60,54 +55,86 @@ public JackFiveBallAutoCommand(DrivebaseSubsystem drivebaseSubsystem, IntakeSubs
-                        new SequentialCommandGroup(
-                                new Follow2910TrajectoryCommand(drivebaseSubsystem, trajectory1),
-                                new ParallelDeadlineGroup(
-                                        new WaitCommand(1.5),
-                                        new IndexShootCommand(indexSubsystem, localizer),
-                                        new ShooterTargetCommand(shooterSubsystem, localizer, () -> true)),
-                                new Follow2910TrajectoryCommand(drivebaseSubsystem, trajectory2),
-                                new ParallelDeadlineGroup(
-                                        new WaitCommand(1),
-                                        new IndexShootCommand(indexSubsystem, localizer)),
-                                new InstantCommand(
-                                        () -> new ShooterTargetCommand(shooterSubsystem, localizer, () -> false)),
-                                new Follow2910TrajectoryCommand(drivebaseSubsystem, trajectory3),
-                                new WaitCommand(2),
-                                new Follow2910TrajectoryCommand(drivebaseSubsystem, trajectory4),
-                                new ParallelDeadlineGroup(
-                                        new WaitCommand(3),
-                                        new IndexShootCommand(indexSubsystem, localizer),
-                                        new ShooterTargetCommand(shooterSubsystem, localizer, () -> true)))));
+                        manager.disableOn(0),
+                        new IntakeSetStopCommand(intakeSubsystem)));
+
+        //
+        // addCommands(
+        // new IntakeSetExtendCommand(intakeSubsystem),
+        // new InstantCommand(() -> new ShooterTargetCommand(shooterSubsystem, localizer, () -> false)),
+        // new ParallelCommandGroup(
+        // new IntakeCommand(intakeSubsystem, indexSubsystem),
+        // new SequentialCommandGroup(
+        // new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_1),
+        // new ParallelDeadlineGroup(
+        // new WaitCommand(1.5),
+        // new IndexShootCommand(indexSubsystem, localizer),
+        // new ShooterTargetCommand(shooterSubsystem, localizer, () -> true)),
+        // new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_2),
+        // new ParallelDeadlineGroup(
+        // new WaitCommand(1),
+        // new IndexShootCommand(indexSubsystem, localizer)),
+        // new InstantCommand(
+        // () -> new ShooterTargetCommand(shooterSubsystem, localizer, () -> false)),
+        // new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_3),
+        // new WaitCommand(2),
+        // new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_4),
+        // new ParallelDeadlineGroup(
+        // new WaitCommand(3),
+        // new IndexShootCommand(indexSubsystem, localizer),
+        // new ShooterTargetCommand(shooterSubsystem, localizer, () -> true)))));
```

Comment:
Once this is tested, please delete the commented-out code before completing the PR.